assets/js/index.js

```diff
@@ -94,7 +94,7 @@ Source:
 
   // https://discourse.gohugo.io/t/range-length-or-last-element/3803/2
 
-  {{ $list := (where .Site.Pages "Section" "docs") -}}
+  {{ $list := .Site.Pages -}}
```

Comment:
This fixes the bug seen in https://github.com/BluBracket/docs.blubracket.com/pull/53#issuecomment-1035192775